planner/core/casetest/physical_plan_test.go

```diff
@@ -2586,3 +2586,24 @@ func TestIndexMergeOrderPushDown(t *testing.T) {
 		tk.MustQuery("explain format = 'brief' " + ts).Check(testkit.Rows(output[i].Plan...))
 	}
 }
+
+// It's a case for index merge's order prop push down.
+func TestIssue43178(t *testing.T) {
```

Comment:
casetest is to put the test which uses data file to test. I think this test can move into ```planner/issuetest``` like ```executor/issuetest```.